docs/rfcs/0001-high-level-architecture.md

```diff
@@ -23,9 +23,9 @@
 
 **Creation Date**: `2020-12-10`
 
-**Last Updated**: `2020-12-10`
+**Last Updated**: `2020-06-08`
 
-**Version Number**: `v1.0.1`
+**Version Number**: `v1.1.0`
```

Comment:
this much was a bit unexpected maybe, but I'm fine with it I guess, as most of the changes (e.g. removing two values) were already done in v1.0.1

Reply:
Yep, the previous PR was a bit rushed (last updated date wasn't updated either), so with all the changes this should already be v1.1.0.